include/mc_rbdyn/JointSensor.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2015-2019 CNRS-UM LIRMM, CNRS-AIST JRL
+ */
+
+#pragma once
+
+#include <mc_rbdyn/Device.h>
+#include <mc/rtc/deprecated.hh>
+
+#include <Eigen/StdVector>
+
+namespace mc_rbdyn
+{
+
+/** This structure defines a joint sensor that provides
+ * temperature and current information for a specfic joint. */
+struct MC_RBDYN_DLLAPI JointSensor : public Device
+{
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+  /** Default constructor, does not represent a valid joint sensor */
+  inline JointSensor() : JointSensor("", "") {}
+
+  /** Constructor
+   *
+   * @param name Name of the sensor
+   *
+   * @param jointName Name of the joint to which the sensor is attached
+   *
+   */
+  inline JointSensor(const std::string & name, const std::string & jointName) : Device(name, jointName)
```

Comment:
This breaks with the convention of other devices which are attached to body rather than joints. Maybe we don't need to have this sensor derive from `Device`?